utils/sl/components/__init__.py

```diff
@@ -287,7 +287,8 @@ def define_components(reqs):
                 commands=['git clean -dxf ',
                           './autogen.sh',
                           './configure --prefix=$ARGOBOTS_PREFIX CC=gcc'
-                          ' --enable-valgrind',
+                          ' --enable-valgrind'
+			  ' --enable-stack-unwind',
```

Comment:
Are you using tabs here?  When I select this line, it looks like tabs...It should be spaces in python file

Reply:
Oops, good catch and thanks !